sdk/helper/keysutil/policy.go

```diff
@@ -1532,11 +1545,13 @@ func (p *Policy) RotateInMemory(randReader io.Reader) (retErr error) {
 	entry.HMACKey = hmacKey
 
 	switch p.Type {
-	case KeyType_AES128_GCM96, KeyType_AES256_GCM96, KeyType_ChaCha20_Poly1305:
+	case KeyType_AES128_GCM96, KeyType_AES256_GCM96, KeyType_ChaCha20_Poly1305, KeyType_HMAC:
 		// Default to 256 bit key
 		numBytes := 32
 		if p.Type == KeyType_AES128_GCM96 {
 			numBytes = 16
+		} else if p.Type == KeyType_HMAC && p.KeySize > 32 {
```

Comment:
16 is the wrong minimum, it should be 32 everwhere for 256 bit.  We're trying to prevent it from being less than the minimum.  We could have 0 == 32 but if the default changes the key size would change on rotation.